src/main/java/mocha/game/world/entity/movement/RandomMovement.java

```diff
@@ -0,0 +1,21 @@
+package mocha.game.world.entity.movement;
+
+import mocha.game.world.Location;
+
+/**
+ * Created by ian on 7/17/17.
+ */
```

Comment:
I think I ended up Implementing this later. If you pull the latest changes and submit another pull request using the current RandomMovement, I think you'll be in the clear.